app/models/answer.rb

```diff
@@ -1,40 +1,36 @@
 class Answer < ApplicationRecord
-  ANSWER_TYPES = {
-    positive: 1,
-    neutral: 2,
-    negative: 3
-  }
   DEFAULT_VALUES = {
     positive: 'Yes',
     neutral: 'Undecided',
     negative: 'No'
   }
 
-  belongs_to :debate
+  enum answer_type: { positive: 1, neutral: 2, negative: 3 }
```

Comment:
I left it this way for backwards compatibility for now.

When enum is set with array of symbols then values start from `0`:

```ruby
Answer.answer_types
#=> {"positive"=>0, "neutral"=>1, "negative"=>2}
```
will check if nothing uses this values directly and change if it is possible.